core/tests/protractor_desktop/explorationTranslationTab.js

```diff
@@ -204,7 +204,6 @@ describe('Exploration translation and voiceover tab', function() {
         await forms.toRichText('Sample Translation.'));
       await explorationEditorPage.saveChanges('Adds one translation.');
       explorationEditorTranslationTab.expectFeedbackTabToBeActive();
-      await workflow.publishExploration();
```

Comment:
Why is this deleted (and, if that is intentional, why are we asserting the same thing twice)?